### DISCUSSION TITLE
I can't merge branches in git

### DISCUSSION THREAD

**Discussion by LuisMatheus-dev** [community/community#23801] [New to GitHub]
<p>I have a repository, to organize its structure, I created a folder and moved my files there and added the change to my master-backup <strong>(secondary)</strong> branch, I did<span> </span><strong>$git merge</strong> for the union with my <strong>(main)</strong> master, but the changes did not are applied, only my subfolder is empty ,and no errors are returned, only that</p>
<pre><code>Everything up-to-date
</code></pre>
<p>What I want, is to add the changes, (which is the folder where I moved my files) to my remote master branch.</p>
<p>In the current situation, my master-backup has the files while the master is empty</p>
<p>How should I proceed?</p>

**Comment by yamiacat** [comment, marked as answer]
<p>When you use <code>git merge</code>, you need to specify which branch is going to get merged in, and it looks like you may have missed this step?</p>
<p>First make sure you’re on the <code>master</code> branch:</p>
<p><code>git checkout master</code></p>
<p>Then merge in the branch you want:</p>
<p><code>git merge master-backup</code></p>
